lte/gateway/configs/gateway.mconfig

```diff
@@ -30,7 +30,7 @@
       "amf_name": "AMF_1",
       "amfRegionId": "1",
       "amfSetId": "1",
-      "amfPointer": "0",
+      "amfPointer": "3",
```

Comment:
what is the reason for this change?

Reply:
UE is accepting registration accept after changing amfPointer to non zero value